include/simdjson/portability.h

```diff
@@ -41,6 +41,11 @@ static inline int hamming(uint64_t input_num) {
 
 #if defined(__BMI2__) || defined(__POPCOUNT__) || defined(__AVX2__)
 #include <x86intrin.h>
+#elif defined(__SSE4_2__)
```

Comment:
I'd say that's not the right includes... It should still be 

#include <x86intrin.h>